crates/cxx-qt-build/src/lib.rs

```diff
@@ -494,4 +507,68 @@ impl CxxQtBuilder {
         }
         self.cc_builder.compile("cxx-qt-gen");
     }
+
+    fn range_for(error: &Error, source: &str) -> Option<std::ops::Range<usize>> {
```

Comment:
This function is unrelated to the CxxQtBuilder struct. I think it would be better to make it a free function (private to the module).

Reply:
Yes, absolutely, as I mentioned, this is in no way production-ready code.
I just dumped it here to try diagnostics out.